frontend/src/components/form/Checkbox.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { toRef } from 'vue';
+import { useField, ErrorMessage } from 'vee-validate';
+
+import { Checkbox } from '@/lib/primevue';
+
+// Props
+type Props = {
+  helpText?: string;
+  label?: string;
+  name: string;
+  disabled?: boolean;
+  bold?: boolean;
+};
+
+const props = withDefaults(defineProps<Props>(), {
+  helpText: '',
+  type: 'text',
+  label: '',
+  disabled: false,
+  bold: true
+});
+
+const { errorMessage, value } = useField<string>(toRef(props, 'name'));
+</script>
+
+<template>
+  <div class="field col">
+    <div class="flex align-items-center">
```

Comment:
Not completely certain, but you might be able to collapse these two divs together into one class with all four class attributes and still have it behave the same.